images/runtime/3.7.1/Dockerfile

```diff
@@ -1 +1,3 @@
 FROM python:3.7.1-slim-stretch
+
+RUN addgroup -g 1000 -S fn && adduser -S -u 1000 -G fn fn
```

Comment:
@denismakogon should we move this image to https://github.com/fnproject/dockers ? fdks should be code only? at least, we're not consistent at the moment, and a few reside in the dockers repo already (python3.6 even)

Reply:
additional point is that the dockers repo has a handy script for updating the lang images (would like to add this to ci to automate even, haven't quite gotten time yet)

Reply:
the reason for keeping image files here is that dockers repo looks more like a dump of whatever we had from IronFunctions.

I'm not sure that moving files back there is a good idea unless the repo would be cleaned up.

Reply:
regarding image files, only fdk-go didn't have its image files in the fdk repo, but the rest keeping their files in place, but not in dockers.

Reply:
FDKs should own their images. Dockers repo should probably die.

Reply:
@rikgibson rationale? now that we've convoluted the fdk contract so much (uds, fn user), I'm not sure I disagree, as most of the other images now in dockers are relatively useless without accompanying fdk code. point on tooling to update stands, each fdk would be responsible for keeping its images up to date. additional point that it conflates fdk with fdk image (the 'runtime image' push failed) when the reality is they aren't necessarily related, that is, a user could use an fdk without the fdk image (separate altogether, not just based on) and it could work.

Reply:
Can we have this conversation independently of this PR and not block what @nishalad95 needs to do?

Reply:
sure, not blocking - there's a python image to update in dockers repo, as well, it's unclear which is the preferred place (one is 3.6 and one is 3.7) and this is a fine excuse to find such things and straighten them out

Reply:
ah, I see - in that case I say bring the 3.6 image in here - the java FDKs builds 8, 11 and SVM images.

Reply:
sorry, pay attention rik...

Reply:
as to rationale, I think FDKs should own their build images, runtime images, init images and maybe a test/hello-world image, as one can't simply use docker/hello-world anymore...

Reply:
So, I'm going to merge this PR.
This is quite a wider discussion regarding where image files should live, but it's just out of the scope of this PR.